### ISSUE TITLE
What's the point using first 10,000 points in training set for training, and last 5,000 points in test set for evaluation?

### ISSUE THREAD

**Issue by ch3cook-fdu** [alexzhou907/PVD#11]

**Comment by Anye666** [comment]
I have same question.
